packages/dx-react-grid/docs/reference/table-summary-row.md

```diff
@@ -0,0 +1,125 @@
+totalRowComponent | ComponentType&lt;[Table.RowProps](table.md#tablerowprops)&gt; | | A component that renders a row with total summaries.
+groupRowComponent | ComponentType&lt;[Table.RowProps](table.md#tablerowprops)&gt; | | A component that renders a row with group summaries.
+treeRowComponent | ComponentType&lt;[Table.RowProps](table.md#tablerowprops)&gt; | | A component that renders a row with tree summaries.
+totalCellComponent | ComponentType&lt;[TableSummaryRow.CellProps](#tablesummaryrowcellprops)&gt; | | A component that renders a cell with total summaries.
+groupCellComponent | ComponentType&lt;[TableSummaryRow.CellProps](#tablesummaryrowcellprops)&gt; | | A component that renders a cell with group summaries.
+treeCellComponent | ComponentType&lt;[TableSummaryRow.CellProps](#tablesummaryrowcellprops)&gt; | | A component that renders a cell with tree summaries.
+treeColumnCellComponent | ComponentType&lt;[TableSummaryRow.CellProps](#tablesummaryrowcellprops)&gt; | | A component that renders a tree cell within a tree summary row.
+treeColumnContentComponent | ComponentType&lt;[TableSummaryRow.ContentProps](#tablesummaryrowcontentprops)&gt; | | A component that renders a tree cell's content.
+treeColumnIndentComponent | ComponentType&lt;[TableSummaryRow.IndentProps](#tablesummaryrowindentprops)&gt; | | A component that renders an indent used to identify a tree row level.
+itemComponent | ComponentType&lt;object&gt; | | A component that renders a summary item.
+messages? | [TableSummaryRow.LocalizationMessages](#localization-messages) | | An object that specifies localization messages.
+
+## Interfaces
+
+### TableSummaryRow.CellProps
+
+Describes properties passed to a component that renders a cell within a row.
+
+Extends [Table.CellProps](table.md#tablecellprops)
+
+Field | Type | Description
+------|------|------------
+column | [Column](grid.md#column) | Specifies the cell's column.
+children? | ReactNode | A React node to be rendered within the cell.
+
+### TableSummaryRow.ContentProps
+
+Describes properties passed to a component that renders a cell's content.
+
+Field | Type | Description
+------|------|------------
+children? | ReactNode | A React node to be rendered within the cell's content.
+
+### TableSummaryRow.IndentProps
+
+Describes properties passed to a component that renders an indent used to identify a row level.
+
+Field | Type | Description
+------|------|------------
+level | number | Specifies the row level.
+
+## Localization Messages
+
+Field | Type | Default | Description
+------|------|---------|------------
+count? | string | 'Count' | Specifies the 'count' summary text.
+sum? | string | 'Sum' | Specifies the 'sum' summary text.
+min? | string | 'Min' | Specifies the 'min' summary text.
+max? | string | 'Max' | Specifies the 'max' summary text.
+avg? | string | 'Avg' | Specifies the 'avg' summary text.
```

Comment:
Maybe in the description, we should explain more?
For example, `Specifies the text for average type`